Libraries/Components/Picker/Picker.js

```diff
@@ -15,6 +15,7 @@
 var ColorPropType = require('ColorPropType');
 var PickerIOS = require('PickerIOS');
 var PickerAndroid = require('PickerAndroid');
+var PickerUbuntu = require('PickerUbuntu');
```

Comment:
If implemented Picker control is not intended to be used only on Ubuntu, it makes sense to remove `Ubuntu` from control title. Maybe, `PickerQt` should work better?

Reply:
@vkjr I would prefer to make separate commit to change `ubuntu` platform to something else, but all others changes (like renaming of exported classes, etc) better to make in separate commits also. It should be more easy to fix broken things which should occur with high possibility :)
Changing of `ubuntu` platform evolves updating of JS code to init and build react-native apps. This brings additional complexity.

Reply:
@MaxRis, sure, agree about different git commits :) I used incorrect words.
What I mean is that if we decide to rename `SomethingUbuntu` to `SomethingQt` in JS code, then it would be nice to do similar changes as a one activity (and probably different commits).
Quick search shows that there is also `UbuntuPage` and `UbuntuScrollView`. I think it would be nice to rename them too.
Of course it is not a big deal to have `SomethingUbuntu` and `SomethingQt` in code simultaneously, it just "smells" worse, as for me :) 

Added #125 to not forget this chages.